arch/arc/core/isr_wrapper.S

```diff
@@ -306,6 +306,16 @@ _skip_sys_power_save_idle_exit:
 SECTION_FUNC(TEXT, _isr_demux)
 	push_s r3
 
+/* according to ARCv2 ISA, r25, r30, r58, r59 are caller-saved
+ * scratch registers, possibly used by interrupt handlers
```

Comment:
Please remove the comments on accl/acch NOT being handled at the top of this file, it's handled now :-)

Reply:
Fixed